web/pdf_page_view.js

```diff
@@ -104,7 +104,13 @@ const LAYERS_ORDER = new Map([
   ["textLayer", 1],
   ["annotationLayer", 2],
   ["annotationEditorLayer", 3],
-  ["xfaLayer", 2],
+  ["xfaLayer", 3],
+]);
+
+const XFA_LAYERS_ORDER = new Map([
+  ["canvasWrapper", 0],
+  ["xfaLayer", 1],
+  ["annotationLayer", 2],
```

Comment:
Are these changes actually necessary, or were the test failures fixed by the changes in `web/annotation_editor_layer_builder.js`?